packages/select/src/components/select/suggest.tsx

```diff
@@ -38,6 +38,12 @@ export interface ISuggestProps<T> extends IListItemsProps<T> {
     /** Custom renderer to transform an item into a string for the input value. */
     inputValueRenderer: (item: T) => string;
 
+    /**
+     * Optionally, if provided the selected item will be in controlled mode.
+     * Use the onItemSelect function to monitor changes.
+     */
+    selectedItem?: T;
```

Comment:
make this `T | null` so `null` can indicate "controlled empty selection"

Reply:
Now it supports the controlled empty selection. I've also added a tests for this specific scenario.

Reply:
@giladgray does it look to you?